app/controllers/organisations_controller.rb

```diff
@@ -133,7 +134,17 @@ def update
         else
           flash[:notice] = I18n.t("organisation.updated")
         end
+        existing_rent_periods = @organisation.organisation_rent_periods.map(&:rent_period)
+        rent_periods_to_create = selected_rent_periods.map(&:to_i) - existing_rent_periods
```

Comment:
Could we do `find_or_create_by` when creating the new periods instead?

Reply:
We probably could but I'm not really concerned about that.
I put a uniqueness validation in the model anyway, so I could just call create on them all and let the duplicate inserts fail quietly.
I'm more slightly annoyed to be getting things from the db before doing this anyway, which feels like it should be unnecessary. I could just pass all values through from the form in a hidden field and and delete all `OrganisationRentPeriod.where(org: @org, rent_period: [unchecked_vals])`
What are your thoughts on that?